ExcelMakerConsole/ExcelFileMaker.cs

```diff
@@ -121,7 +102,10 @@ private string GetExcelColumnName(int columnNumber)
 
             return columnName;
         }
-
+        private bool datasets_have_face_force(List<Dataset> datasets)
+        {
+            return datasets.Count == 6;

```

Comment:
this is really fragile.. is there a better way? I don't remember what the job file looks like, but is there by chance a header on the csv we can look at or something?